docs/snippets/defaults_channel_mapping.mdx

```diff
@@ -35,4 +35,43 @@ default_platform_ad_format_by_channel:
   audio:    30s Audio
   app:      Interstitial - 1080x1920 Banner
   web:      Leaderboard - 728x90 Banner
+default_platform_ad_format_by_channel:
```

Comment:
Is this duplicative? 
<img width="519" alt="image" src="https://github.com/scope3data/methodology/assets/2388660/ab06968b-38c9-479a-9c0b-9206721688da">